src/Illuminate/Database/Query/Grammars/PostgresGrammar.php

```diff
@@ -517,4 +546,22 @@ protected function wrapJsonPathAttributes($path)
                         : "'$attribute'";
         }, $path);
     }
+
+    /**
+     * Binds parameter to query.
+     *
+     * @param  Builder  $query
+     * @param  mixed  $value
+     * @return mixed
+     */
+    protected function bindParameter(Builder $query, $value)
+    {
+        if ($this->isExpression($value)) {
+            return $this->getValue($value);
+        }
+
+        $query->addBinding($value);
```

Comment:
Same binding order issue here. Bindings must be added while the query is being built otherwise they will never be in the correct order?

Reply:
What should then be the approach to fixing this bug?

Reply:
@humaneguy see the convo above.